tre_schemas/avro/request-courtDocument-parse.avsc

```diff
@@ -0,0 +1,60 @@
+{
+  "type": "record",
+  "name": "RequestCourtDocumentParse",
+  "namespace": "uk.gov.nationalarchives.tre.messages.request.courtdocument.parse",
+  "doc": "properties required for all messages routed through TRE",
+  "fields": [
+    {
+      "name": "properties",
+      "type": "uk.gov.nationalarchives.tre.messages.event.Properties"
+    },
+    {
+      "name": "parameters",
+      "doc": "message specific parameters required by the consumer to be be able to process event",
+      "type": {
+        "type": "record",
+        "name": "Parameters",
+        "fields": [
+          {
+            "name": "s3Bucket",
+            "type": "string",
+            "doc": "s3 bucket containing the document to be parsed"
+          },
+          {
+            "name": "s3Key",
+            "type": "string",
+            "doc": "s3 key of the document to be parsed"
+          },
+          {
+            "name": "reference",
+            "type": "string",
+            "doc": "Opaque FCL-generated reference to allow matching of response to request"
+          },
+          {
+            "name": "originator",
+            "type": [
+              "string",
+              "null"
+            ],
+            "doc": "who provided the docx such as FCL|TDR. This will be passed through to the judgmentpacakage-available message"
+          },
+          {
+            "name": "parserInstructions",
+            "doc": "Parameters to be passed to the document parser",
+            "type": {
+              "type": "record",
+              "name": "ParserInstructions",
+              "fields": [
+                {
+                  "name": "documentType",
```

Comment:
we will need to allow this to be null for TDR

Reply:
the parse request will not be sent by TDR. TDR will keep using a message indicating a new transfer is available.
Do you want the option to have null?

Reply:
Oh interesting. For my own understanding, what would the difference between these 2 requests (FCL vs TDR) be?

If it's coming from us, we shouldn't need to since we'll be able to deduce the type easily.